domain/modelagent/state/state.go

```diff
@@ -75,3 +75,54 @@ WHERE uuid = $M.model_id
 	}
 	return vers, nil
 }
+
+// AgentVersionForModelName returns the agent version for the model with the
+// given name and owner.
+func (st *State) AgentVersionForModelName(
```

Comment:
We should mention all the possible errors returned from this function in the comment.